packages/web-runtime/src/components/Topbar/TopBar.vue

```diff
@@ -216,11 +217,8 @@ export default {
   }
 
   .oc-topbar-left {
-    gap: 30px;
-
-    img.oc-logo-image {
-      height: 38px;
-    }
+    width: 10rem;
```

Comment:
Why do you need the `width`? Value seems odd..

Reply:
i think its dev leftover